quic/state/stream/StreamClosedHandlers-inl.h

```diff
@@ -76,9 +76,9 @@ inline void Handler<
     StreamSendStates::Closed,
     StreamEvents::SendReset>::
     handle(
-        QuicStreamState::Send& /*state*/,
+        FOLLY_MAYBE_UNUSED QuicStreamState::Send& state,
         StreamEvents::SendReset&&,
-        QuicStreamState& /*stream*/) {
```

Comment:
Internally we actually prefer using the `/*param*/` style versus `FOLLY_MAYBE_UNUSED` for function parameters. Could you change these back or did that cause warnings?

Reply:
Sure, can do.  I was just refreshing this PR since it hadn't been merged yet.

I'm currently running into another problem with the latest boost and fizz, though.  Are there any known issues with boost 1.70 and fizz, perchance?

Reply:
Ah, yes, there *is* a problem with fizz and boost v1.70:

* https://github.com/facebookincubator/fizz/issues/25, and its dupes:
    * https://github.com/facebookincubator/fizz/issues/31
    * https://github.com/facebookincubator/fizz/issues/33

Looks like it is a problem in upstream boost -- v1.69 is apparently ok, and v1.71 is supposed to be ok, too (homebrew currently has 1.70.0).

Reply:
Just in case anyone else runs into this and is in the same boat as me (building on MacOS with homebrew while homebrew still has boost v1.70.0 with this known problem), here's how I solved it:

```sh
# Install boost v1.70
$ brew install boost

$ curl https://patch-diff.githubusercontent.com/raw/boostorg/variant/pull/68.diff > ~/Downloads/boost-68.diff
$ cd /usr/local/include/boost
$ patch -p3 < ~/Downloads/boost-68.diff
```

True, this modifies your currently-installed boost.  YMMV.

Reply:
@mjoras Question: what is the recommended style for a parameter that *might* be unused?  For example, [`cmsg` will be unused here](https://github.com/facebookincubator/mvfst/blob/b9b63e527bdd973f414ddab9a8f5b7062575725f/quic/client/QuicClientTransport.cpp#L959-L980) only `#if !defined(FOLLY_HAVE_MSG_ERRQUEUE)`.  

Clearly, commenting out the parameter is not good enough.  Should I use `FOLLY_MAYBE_UNUSED` here?